.editorconfig

```diff
@@ -0,0 +1,18 @@
+# EditorConfig helps developers define and maintain consistent
+# coding styles between different editors and IDEs
+# editorconfig.org
+
+root = true
+
+[*]
+
+indent_style = space
```

Comment:
We prefer tabs over spaces (allows an end user to configure the width of indentation that they prefer to see). Other than with package.json where we must conform to npm.